src/mame/drivers/stlforce.cpp

```diff
@@ -70,6 +70,9 @@ TO DO :
 */
 
 #include "emu.h"
+
+#include "emupal.h"
+#include "video/edevices.h"
 #include "includes/stlforce.h"
```

Comment:
Please `#include` your module header `stlforce.h` immediately after `emu.h` and before other headers - this helps ensure that headers `#include` device headers they need.